environments/sbox.tfvars

```diff
@@ -21,4 +21,22 @@ frontends = [
     certificate_name = "STAR-sandbox-platform-hmcts-net"
     disabled_rules   = {}
   },
+  {
+    name             = "idam-web-public"
+    custom_domain    = "idam-web-public-aks.sandbox.platform.hmcts.net"
```

Comment:
there needs to be a PR to azure-public-dns to set up the domain validation,

https://github.com/hmcts/azure-public-dns/blob/master/environments/sandbox.yml#L52-L58

and also a cdnverify for the shuttering:
https://github.com/hmcts/azure-public-dns/blob/master/environments/prod/track-benefit-appeal-service-gov-uk.yml#L30-L32

Reply:
Not exactly sure how you need this. Can you provide any more instructions?

Reply:
Azure front door and the azure cdn we use for shuttering both require domain ownership proof, we always use the indirect validation because otherwise if someone shutters an app we can’t deploy changes to front door as it checks that you own the domain every time you make changes to it. 

Is there a specific bit you aren’t sure about?